qa/workunits/rados/test_crash.sh

```diff
@@ -26,7 +26,7 @@ done
 
 # let daemon find crashdumps on startup
 sudo systemctl restart ceph-crash
-sleep 30
+sleep 60
```

Comment:
How reproducible is this https://tracker.ceph.com/issues/58098? We should try to understand why the existing sleep value isn't enough any longer so as to not mask other issues with slow OSD restarts.

Reply:
I'll run some tests to try and reproduce the issue, and mark this PR as a draft for now.